tests/sentry/nodestore/bigtable/backend/tests.py

```diff
@@ -75,19 +109,19 @@ def ns(request):
     ids=["zlib", "ident", "zstd"],
 )
 def test_get(ns, compression, expected_prefix):
-    ns.compression = compression
+    ns.store.compression = compression
     node_id = "node_id"
     data = {"foo": "bar"}
     ns.set(node_id, data)
 
     # Make sure this value does not get used during read. We may have various
     # forms of compression in bigtable.
-    ns.compression = lambda: 1 / 0
+    ns.store.compression = lambda: 1 / 0
     # Do not use cache as that entirely bypasses what we want to test here.
     ns.cache = None
     assert ns.get(node_id) == data
```

Comment:
This should probably be moved down an abstraction layer to test the `BigtableKVStorage` implementation to avoid this kind of thing.